cmd/talosctl/cmd/mgmt/gen/config.go

```diff
@@ -170,9 +170,6 @@ func GenerateConfigBundle(genOptions []generate.Option,
 		return nil, fmt.Errorf("failed to generate config bundle: %w", err)
```

Comment:
I guess we also need a error message if no endpoints is set, unless it's  already there?

Reply:
we don't need it, it's fine to have no endpoints (`talosctl` will complain when such `talosconfig` is used). 

from the point of view of generation, it's fine to have no endpoints. it just drops unfortunate default